.identity/github_environment_web_apps.tf

```diff
@@ -0,0 +1,35 @@
+resource "github_repository_environment" "messages_sending" {
```

Comment:
Please add another folder under `infra/repository` with these resources. [Here](https://github.com/pagopa/dx-typescript/tree/main/infra/repository) an example you can almost copy and paste.

P.S. I'm working on a module to make easier this setup